### PR TITLE
{vis}[intel/2017a] R v3.3.3, libGLU v9.0.0, Tk v8.6.6, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
depends on ~~#4416~~, ~~#4423~~, ~~#4425~~, ~~#4428~~, ~~#4429~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2009.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/260e2b5406ea6be51482a2ebf97f05ce for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2411.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dac0d2c23fd22c21e054b587505f16ab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/31b771a9727ca30ca9fcbf6ec4ea74b2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2485.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f09748c57b52a85681db3be5a5ecf6c7 for a full test report.

File: easybuild/easyconfigs/r/R/R-3.3.3-intel-2017a-X11-20170314.eb [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+    ('pim', '2.0.0.2', ext_options),
+    ('minpack.lm', '1.2-1', ext_options),
+    ('rootSolve', '1.7', ext_options),
+    ('diagram', '1.6.3', ext_options),
+    ('FME', '1.3.5', ext_options),
+    ('bmp', '0.2', ext_options),
+    ('readbitmap', '0.1-4', ext_options),
+    ('purrr', '0.2.2', ext_options),
+    ('downloader', '0.4', ext_options),
+    ('imager', '0.40.1', dict(ext_options.items() + [('patches', ['imager-0.40.1_fix-CImg-icpc.patch'])])),
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2017a-X11-20170314.eb]: @hajgato I was forced to update `imager` to 0.40.1 (and include `downloader` as well) + apply a patch to make this build with Intel compilers

Shall we update `imager` in the existing R v3.3.3 easyconfig that uses `foss/2016b` as well to sync it up?

Comment (hajgato) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2017a-X11-20170314.eb]: @boegel At the end it makes sense. I can do it when I am back from hollidays. (18th April 2017).

Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.3.3-intel-2017a-X11-20170314.eb]: done in #4449

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0a17cb5b5d3beb3b10f238382b1080e1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d1fb2a21344f28cd1b06bd33cb64b250 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/51879db898443c7a86f5a9d02fe41ae9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d49319daeac7a42d15e99c44c9469ba2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8e6ea340ca9e4c471027fb2df56adfbc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/98e1faba50b3cf633ae6f9c728bf6e73 for a full test report.

**Comment by boegel** [comment]
other than the `imager` update (synced in #4449), just a toolchain bump (with updated dependencies)

**Pull request merged**